install_config/configuring_sdn.adoc

```diff
@@ -183,6 +192,27 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
+[[growing-the-service-network]]
+== Growing the Service Network
+
+If you are running low on addresses in your Service network and can expand the range so that it includes the current range at the beginning of the new range, then you can:
+
+. Change the `*serviceNetworkCIDR*` parameter on all
+xref:configuring-the-pod-network-on-masters[masters] in their configuration files.
+ifdef::openshift-origin[].  You can only change the number following the / to a smaller number.
+. Delete the `*clusterNetwork*` object in OpensShift:
+----
+$ oc delete clusternetwork default
+----
+. Restart the *origin-master-api* and *origin-master-controller*  services on masters .
+endif::[]
+ifdef::openshift-enterprise[]
+. Restart the *atomic-openshift-master-api* and *atomic-openshift-master-controllers*
+on masters.
+endif::[]
+. Update the inventory file to change the openshift_portal_net to the new CIDR and run the playbook `*config.yml*` to redeploy the cluster
+. Evacuate each node, one by one, restart it, and then re-enable it
```

Comment:
There's a section in the day two that I've been meaning to create an include for so that it can be used in places like this:

https://docs.openshift.com/container-platform/3.9/day_two_guide/docker_tasks.html#evacuating-the-node

I can throw this in to this procedure after merging, maybe.